CREDITS

```diff
@@ -116,12 +116,13 @@ of the Coq Proof assistant during the indicated time:
   Matej Košík (INRIA, 2015-2017)
   Leonidas Lampropoulos (University of Pennsylvania, 2018)
   Pierre Letouzey (LRI, 2000-2004, PPS, 2005-2008,
-                   INRIA-PPS then IRIF, 2009-now)
```

Comment:
Does this Fehrle character deserve a mention?

Reply:
Sorry, I am not sure I understood your question? If you mean that you should be listed in there, yes, but I've generally renounced to listing everyone (at some point adding oneself was part of the recommendations of the contributing guide, but it was so little followed that I dropped it in the new version, and that I am looking for a better way to definitely replace this CREDITS file).

Reply:
OK.  By the way "given up on" would be better and more common than "generally renounced to".  Renounce = "to give up, refuse, or resign usually by formal declaration".

Reply:
Thanks for the feedback. Although, in this precise case I almost added "officially" in front of "renounce" so it turns out not so wrong after all :laughing: